cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -3,6 +3,8 @@
 
 set -exuo pipefail
 
+(cd ../gocli && make container)
```

Comment:
We can change working directory with make `make -C ../gocli container`

Reply:
Done.